master/docs/guides/svn.rst

```diff
@@ -0,0 +1,92 @@
+=================================
+Subversion Version Control System
```

Comment:
> these are each focused on a particular usage topic that might involve content from all over the documentation (particularly for version-control systems, which will reference both change sources and source steps).

This is exactly what I have in mind.

Let's return to the topic of title when I think this "guide" is ready.